internal/pkg/levee/levee.go

```diff
@@ -122,3 +203,25 @@ func getArgumentPropagator(c *config.Config, call *ssa.Call) ssa.Node {
 
 	return nil
 }
+
+// callValues collects the ssa.Values that are arguments to an ssa.Call.
+// in particular, it handles variadic arguments to some extent.
+// We don't want to analyze what the variadic argument points to (a slice), but we do
+// want to analyze the values *within* the slice.
+func callValues(c *ssa.Call) (values []ssa.Value) {
+	for _, a := range c.Call.Args {
+		slice, ok := a.(*ssa.Slice)
+		if !ok {
+			values = append(values, a)
+			continue
+		}
+		refs := *slice.X.Referrers()
```

Comment:
Good question. I am removing the `varags` package entirely in PR #48. Given that, I think it would be preferable not to, and I don't think this code deserves its own package.